### ISSUE TITLE
Non-ASCII characters in Julia

### ISSUE THREAD

**Issue by Haksell** [codewars/codewars.com#2278]
**Is your feature request related to a problem? Please describe.**
The Julia programming language uses a lot of non-ASCII characters to represent common mathematical functions. It would be nice to be able to generate them directly from the website, instead of having to keep a Julia REPL on the side.
Considering the size of the Julia userbase on codewars, I realise that it is probably not a very crucial feature request.

**Describe the solution you'd like**
I would like the same functionality as in a Julia REPL or notebook. When I type `\sqrt` followed by `<TAB>`, it should replace what I typed with `√`.

**Comment by kazk** [comment]
> Considering the size of the Julia userbase on codewars, I realise that it is probably not a very crucial feature request.

We can create a community maintained CodeMirror extension. We've made one for Agda in the past https://github.com/codewars/codemirror-agda which you can try [here](https://codewars.github.io/codemirror-agda/).

Is there a list of symbols supported by Julia REPL? Agda one is based on its Emacs mode if I remember correctly, and the names are probably different or missing characters often used in Julia.

**Comment by Haksell** [comment]
Yes, here is the full list: https://docs.julialang.org/en/v1/manual/unicode-input/.

**Comment by kazk** [comment]
We can't support them all (3326 sequences, 2497 excluding names with colon). I made a CSV from that page https://gist.github.com/kazk/aa5ac312a5ee95210ffb0fa460900672

We'll reduce that to 100 most frequently used ones or something, then write a simple CodeMirror extension.

**Comment by Haksell** [comment]
There are only 26 special characters built into the language: `÷ π ℯ ∈ ∉ ∋ ∌ ∘ √ ∛ ∩ ∪ ≈ ≉ ≠ ≡ ≢ ≤ ≥ ⊆ ⊇ ⊈ ⊉ ⊊ ⊋ ⊻`. Some others that I can imagine useful are some greek letters like `ϵ`, `σ`, `λ` or `Δ`. Subscripts and superscripts might also be nice (`\_1` -> `₁` and `\^3` -> `³`).

I've made a Google form to ask what would be the most-liked characters: https://docs.google.com/forms/d/e/1FAIpQLSer5dWt9kgE-yFXF7qg-O49fRxqgRjGJTrzJkN4fg_A9TfRtQ/viewform?usp=sf_link
Not sure how to contact those who would be interested, though.
